Re-enable Deprecation Checking & Fix Issues

As part of the Drupal 8.9 upgrade we had to disable BLT's deprecated code checking because we were unable to fix the deprecation notices in app_module due to how the core maintainers implemented changes to path_alias. This task is to turn back on the Deprecation checking and fix the deprecation notices.

### Notes
TL;DR; Once we resolve all the deprecation issues we need to turn back on deprecation checking.

Issue #2927 will be updating the check that is done as part of the deprecation validation. We will be adding a new section to disable until we fix all the issues in #3127.  Additionally, in the 8.9 upgrade we added the following, however it looks like the deprecated checks may have been removed in newer BLT versions.

`<repo_root>/blt/ci.blt.yml`
```
disable-targets:
  tests:
    deprecated:
      modules: true
```

**Related Issue:** #2905

## Issues that need to be fixed
```
 ------ ----------------------------------------------------------------------------------------------------------- 
  Line   cgov_blog/src/Services/BlogManager.php                                                                     
 ------ ----------------------------------------------------------------------------------------------------------- 
  88     Parameter $alias_manager of method Drupal\cgov_blog\Services\BlogManager::__construct() has typehint with  
         deprecated interface Drupal\Core\Path\AliasManagerInterface:                                               
         in drupal:8.8.0 and is removed from drupal:9.0.0.                                                          
         Use \Drupal\path_alias\AliasManagerInterface.                                                              
  88     Parameter $entity_query of method Drupal\cgov_blog\Services\BlogManager::__construct() has typehint with   
         deprecated class Drupal\Core\Entity\Query\QueryFactory:                                                    
         in drupal:8.3.0 and is removed from drupal:9.0.0. Use                                                      
         \Drupal\Core\Entity\EntityStorageInterface::getQuery() or                                                  
         \Drupal\Core\Entity\EntityStorageInterface::getAggregateQuery() instead.                                   
 ------ ----------------------------------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------- 
  Line   cgov_core/cgov_core.module                                            
 ------ ---------------------------------------------------------------------- 
  80     Call to deprecated function entity_load():                            
         in drupal:8.0.0 and is removed from drupal:9.0.0. Use the             
         entity type storage's load() method.                                  
  90     Call to deprecated function drupal_set_message():                     
         in drupal:8.5.0 and is removed from drupal:9.0.0.                     
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.  
 ------ ---------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------- 
  Line   cgov_core/src/Plugin/PathProcessor/UrlProcessor.php                                                      
 ------ --------------------------------------------------------------------------------------------------------- 
  56     Parameter $aliasManager of method Drupal\cgov_core\Plugin\PathProcessor\UrlProcessor::__construct() has  
         typehint with deprecated interface Drupal\Core\Path\AliasManagerInterface:                               
         in drupal:8.8.0 and is removed from drupal:9.0.0.                                                        
         Use \Drupal\path_alias\AliasManagerInterface.                                                            
 ------ --------------------------------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------------------------------------------- 
  Line   cgov_core/src/Services/CgovNavigationManager.php                                                          
 ------ ---------------------------------------------------------------------------------------------------------- 
  120    Parameter $pathAliasManager of method Drupal\cgov_core\Services\CgovNavigationManager::__construct() has  
         typehint with deprecated interface Drupal\Core\Path\AliasManagerInterface:                                
         in drupal:8.8.0 and is removed from drupal:9.0.0.                                                         
         Use \Drupal\path_alias\AliasManagerInterface.                                                             
 ------ ---------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------- 
  Line   cgov_event/src/Controller/ICalendarController.php                                                
 ------ ------------------------------------------------------------------------------------------------- 
  51     Call to deprecated method getStorage() of class Drupal\Core\Entity\EntityManager:                
         in drupal:8.0.0 and is removed from drupal:9.0.0.                                                
         Use \Drupal\Core\Entity\EntityTypeManagerInterface::getStorage() instead.                        
  111    Call to deprecated constant FILE_EXISTS_REPLACE: Deprecated in drupal:8.7.0 and is removed from  
         drupal:9.0.0. Use \Drupal\Core\File\FileSystemInterface::EXISTS_REPLACE.                         
 ------ ------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------- 
  Line   cgov_migration/cgov_migration.install                                    
 ------ ------------------------------------------------------------------------- 
  14     Call to deprecated function db_select():                                 
         in drupal:8.0.0 and is removed from drupal:9.0.0. Instead, get           
         a database connection injected into your service from the container and  
         call select() on it. For example,                                        
  16     Call to deprecated function db_like():                                   
         in drupal:8.0.0 and is removed from drupal:9.0.0. Instead,               
         get a database connection injected into your service from the container  
         and call escapeLike() on it. For example,                                
 ------ ------------------------------------------------------------------------- 

 ------ -------------------------------------------------------------------------- 
  Line   cgov_redirect_manager/src/CgovImporterService.php                         
 ------ -------------------------------------------------------------------------- 
  84     Call to deprecated function drupal_set_message():                         
         in drupal:8.5.0 and is removed from drupal:9.0.0.                         
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.      
  89     Call to deprecated function drupal_set_message():                         
         in drupal:8.5.0 and is removed from drupal:9.0.0.                         
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.      
  102    Call to deprecated function drupal_set_message():                         
         in drupal:8.5.0 and is removed from drupal:9.0.0.                         
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.      
  115    Call to deprecated function drupal_set_message():                         
         in drupal:8.5.0 and is removed from drupal:9.0.0.                         
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.      
  216    Call to deprecated function drupal_set_message():                         
         in drupal:8.5.0 and is removed from drupal:9.0.0.                         
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.      
  473    Call to deprecated method entityManager() of class Drupal:                
         in drupal:8.0.0 and is removed from drupal:9.0.0.                         
         Use \Drupal::entityTypeManager() instead in most cases. If the needed     
         method is not on \Drupal\Core\Entity\EntityTypeManagerInterface, see the  
         deprecated \Drupal\Core\Entity\EntityManager to find the                  
         correct interface or service.                                             
 ------ -------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------- 
  Line   cgov_redirect_manager/src/Form/RedirectImport.php                     
 ------ ---------------------------------------------------------------------- 
  121    Call to deprecated function drupal_set_message():                     
         in drupal:8.5.0 and is removed from drupal:9.0.0.                     
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.  
  126    Call to deprecated function drupal_set_message():                     
         in drupal:8.5.0 and is removed from drupal:9.0.0.                     
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.  
 ------ ---------------------------------------------------------------------- 

 ------ ----------------------------------------------------------------------------------------- 
  Line   cgov_site_section/src/Plugin/Validation/Constraint/UniqueCgovUrlConstraintValidator.php  
 ------ ----------------------------------------------------------------------------------------- 
  32     Call to deprecated method getStorage() of class Drupal\Core\Entity\EntityManager:        
         in drupal:8.0.0 and is removed from drupal:9.0.0.                                        
         Use \Drupal\Core\Entity\EntityTypeManagerInterface::getStorage() instead.                
 ------ ----------------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------- 
  Line   cgov_syndication/cgov_syndication.module                              
 ------ ---------------------------------------------------------------------- 
  64     Call to deprecated function entity_load():                            
         in drupal:8.0.0 and is removed from drupal:9.0.0. Use the             
         entity type storage's load() method.                                  
  74     Call to deprecated function drupal_set_message():                     
         in drupal:8.5.0 and is removed from drupal:9.0.0.                     
         Use \Drupal\Core\Messenger\MessengerInterface::addMessage() instead.  
 ------ ---------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------- 
  Line   cgov_vocab_manager/src/Manager/CgovVocabManager.php                                  
 ------ ------------------------------------------------------------------------------------- 
  46     Call to deprecated method getVocabularyId() of class Drupal\taxonomy\TermInterface:  
         in drupal:8.?.? and is removed from drupal:9.0.0. Use                                
         TermInterface::bundle() instead.                                                     
 ------ ------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------- 
  Line   cgov_yaml_content/src/Service/CgovYamlContentEventSubscriber.php                                             
 ------ ------------------------------------------------------------------------------------------------------------- 
  82     Parameter $entityQueryFactory of method                                                                      
         Drupal\cgov_yaml_content\Service\CgovYamlContentEventSubscriber::__construct() has typehint with deprecated  
         class Drupal\Core\Entity\Query\QueryFactory:                                                                 
         in drupal:8.3.0 and is removed from drupal:9.0.0. Use                                                        
         \Drupal\Core\Entity\EntityStorageInterface::getQuery() or                                                    
         \Drupal\Core\Entity\EntityStorageInterface::getAggregateQuery() instead.                                     
  260    Call to deprecated constant FILE_CREATE_DIRECTORY: Deprecated in drupal:8.7.0 and is removed from            
         drupal:9.0.0. Use \Drupal\Core\File\FileSystemInterface::CREATE_DIRECTORY.                                   
  260    Call to deprecated function file_prepare_directory():                                                        
         in drupal:8.7.0 and is removed from drupal:9.0.0.                                                            
         Use \Drupal\Core\File\FileSystemInterface::prepareDirectory().                                               
  263    Call to deprecated constant FILE_EXISTS_REPLACE: Deprecated in drupal:8.7.0 and is removed from              
         drupal:9.0.0. Use \Drupal\Core\File\FileSystemInterface::EXISTS_REPLACE.                                     
 ------ ------------------------------------------------------------------------------------------------------------- 
                                                                                                                 
 [ERROR] Found 26 errors                                                                                               
```

--- a/blt/ci.blt.yml
+++ b/blt/ci.blt.yml
@@ -12,8 +12,3 @@ project.local.hostname: 127.0.0.1:8888
 drush.debug: false
 ## Stop phpunit from trying to write to root.
 reports.localDir: /tmp
-disable-targets:
-  tests:
-    deprecated:
-      modules: true
-      profiles: true

--- a/docroot/modules/custom/app_module/app_module.services.yml
+++ b/docroot/modules/custom/app_module/app_module.services.yml
@@ -7,7 +7,7 @@ services:
     arguments: ['@request_stack']
   app_module.app_path_manager:
     class: Drupal\app_module\AppPathManager
-    arguments: ['@path.alias_manager', '@path.alias_storage', '@app_module.app_path_storage', '@language_manager', '@cache.data']
+    arguments: ['@path_alias.manager', '@entity_type.manager', '@app_module.app_path_storage', '@language_manager', '@cache.data']
   app_module.app_path_storage:
     class: Drupal\app_module\AppPathStorage
     arguments: ['@database', '@module_handler']

--- a/docroot/modules/custom/app_module/src/AppPathManager.php
+++ b/docroot/modules/custom/app_module/src/AppPathManager.php
@@ -6,9 +6,11 @@ use Drupal\Core\Cache\CacheBackendInterface;
 use Drupal\Core\CacheDecorator\CacheDecoratorInterface;
 use Drupal\Core\Entity\ContentEntityInterface;
 use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityMalformedException;
+use Drupal\Core\Entity\Exception\UndefinedLinkTemplateException;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Entity\RevisionableInterface;
-use Drupal\Core\Path\AliasManagerInterface;
-use Drupal\Core\Path\AliasStorageInterface;
+use Drupal\path_alias\AliasManagerInterface;
 use Drupal\Core\Language\LanguageInterface;
 use Drupal\Core\Language\LanguageManagerInterface;
 
@@ -20,17 +22,16 @@ class AppPathManager implements AppPathManagerInterface, CacheDecoratorInterface
   /**
    * The alias manager service.
    *
-   * @var \Drupal\Core\Path\AliasManagerInterface
+   * @var \Drupal\path_alias\AliasManagerInterface
    */
   protected $aliasManager;
 
   /**
-   * The alias manager service.
+   * The Entity Type Manager service.
    *
-   * @var \Drupal\Core\Path\AliasStorageInterface
+   * @var \Drupal\Entity\EntityTypeManagerInterface
    */
-  protected $aliasStorage;
-
+  protected $entityTypeManager;
 
   /**
    * The app path storage service.
@@ -84,9 +85,9 @@ class AppPathManager implements AppPathManagerInterface, CacheDecoratorInterface
   /**
    * Constructs an AppPathManager.
    *
-   * @param \Drupal\Core\Path\AliasManagerInterface $alias_manager
+   * @param \Drupal\path_alias\AliasManagerInterface $alias_manager
    *   The core alias manager service.
-   * @param \Drupal\Core\Path\AliasStorageInterface $alias_storage
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
    *   The core alias storage service.
    * @param \Drupal\app_module\AppPathStorageInterface $storage
    *   The app module path storage service.
@@ -97,13 +98,13 @@ class AppPathManager implements AppPathManagerInterface, CacheDecoratorInterface
    */
   public function __construct(
     AliasManagerInterface $alias_manager,
-    AliasStorageInterface $alias_storage,
+    EntityTypeManagerInterface $entity_type_manager,
     AppPathStorageInterface $storage,
     LanguageManagerInterface $language_manager,
     CacheBackendInterface $cache
   ) {
     $this->aliasManager = $alias_manager;
-    $this->aliasStorage = $alias_storage;
+    $this->entityTypeManager = $entity_type_manager;
     $this->storage = $storage;
     $this->languageManager = $language_manager;
     $this->cache = $cache;
@@ -444,12 +445,67 @@ class AppPathManager implements AppPathManagerInterface, CacheDecoratorInterface
     // alias and call it a day.
     //
     // See \Drupal\path\Plugin\Field\FieldType\PathFieldItemList::computeValue.
-    $path = $this->aliasStorage->load([
-      'source' => $owner_source,
-      'langcode' => $langcode,
-    ]);
+    $path = $this->loadAlias($owner_source, $langcode);
 
     return $path;
+  }
+
+  /**
+   * This loads a path_alias entity.
+   *
+   * Drupal 9 does away with the old AliasStorage class, and there is no
+   * replacement for the old load function. The old function just made an
+   * entity query, so we will copy most of the old code to ensure continued
+   * functionality.
+   *
+   * @param string $path
+   *   The path, which used to be known as source, to load.
+   * @param string $langcode
+   *   The language to fetch.
+   *
+   * @return array|false
+   *   FALSE if the path could not be saved or an associative array containing
+   *   the following keys:
+   *   - source (string): The internal system path with a starting slash.
+   *   - alias (string): The URL alias with a starting slash.
+   *   - pid (int): Unique path alias identifier.
+   *   - langcode (string): The language code of the alias.
+   *   - original: For updates, an array with source, alias and langcode with
+   *     the previous values.
+   */
+  private function loadAlias($path, $langcode) {
+    // We cannot fetch the storage from the EntityTypeManager in the constructor
+    // because it causes a circular dependency in the service container.
+    $aliasStorage = $this->entityTypeManager->getStorage('path_alias');
+
+    $query = $aliasStorage->getQuery();
+    $query->accessCheck(FALSE);
+    $query->condition('path', $path, '=');
+    $query->condition('langcode', $langcode, '=');
+    $result = $query
+      ->sort('id', 'DESC')
+      ->range(0, 1)
+      ->execute();
+
+    if (count($result) > 0) {
+      $entities = $aliasStorage->loadMultiple($result);
+
+      /** @var \Drupal\path_alias\PathAliasInterface $path_alias */
+      $path_alias = reset($entities);
+      if ($path_alias) {
+        return [
+          'pid' => $path_alias
+            ->id(),
+          'source' => $path_alias
+            ->getPath(),
+          'alias' => $path_alias
+            ->getAlias(),
+          'langcode' => $path_alias
+            ->get('langcode')->value,
+        ];
+      }
+    }
+    return FALSE;
   }
 
   /**

--- a/docroot/modules/custom/app_module/tests/src/Unit/AppPathManagerTestBase.php
+++ b/docroot/modules/custom/app_module/tests/src/Unit/AppPathManagerTestBase.php
@@ -15,16 +15,16 @@ abstract class AppPathManagerTestBase extends UnitTestCase {
   /**
    * Alias manager.
    *
-   * @var \Drupal\Core\Path\AliasManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+   * @var \Drupal\path_alias\AliasManagerInterface|\PHPUnit_Framework_MockObject_MockObject
    */
   protected $aliasManager;
 
   /**
    * Alias storage.
    *
-   * @var \Drupal\Core\Path\AliasStorageInterface|\PHPUnit_Framework_MockObject_MockObject
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface|\PHPUnit_Framework_MockObject_MockObject
    */
-  protected $aliasStorage;
+  protected $entityTypeManager;
 
   /**
    * The app path manager.
@@ -55,6 +55,13 @@ abstract class AppPathManagerTestBase extends UnitTestCase {
   protected $cache;
 
   /**
+   * Entity Query Mock.
+   *
+   * @var \Drupal\Core\Entity\Query\QueryInterface|\PHPUnit_Framework_MockObject_MockObject
+   */
+  protected $entityQuery;
+
+  /**
    * The internal cache key used by the alias manager.
    *
    * @var string
@@ -77,10 +84,47 @@ abstract class AppPathManagerTestBase extends UnitTestCase {
     $this->appPathStorage = $this->createMock('Drupal\app_module\AppPathStorageInterface');
     $this->languageManager = $this->createMock('Drupal\Core\Language\LanguageManagerInterface');
     $this->cache = $this->createMock('Drupal\Core\Cache\CacheBackendInterface');
-    $this->aliasManager = $this->createMock('Drupal\Core\Path\AliasManagerInterface');
-    $this->aliasStorage = $this->createMock('Drupal\Core\Path\AliasStorageInterface');
+    $this->aliasManager = $this->createMock('Drupal\path_alias\AliasManagerInterface');
+    $this->entityTypeManager = $this->createMock('Drupal\Core\Entity\EntityTypeManagerInterface');
+    $this->aliasStorage = $this->createMock('Drupal\Core\Entity\EntityStorageInterface');
+    $this->entityQuery = $this->createMock('Drupal\Core\Entity\Query\QueryInterface');
 
-    $this->appPathManager = new AppPathManager($this->aliasManager, $this->aliasStorage, $this->appPathStorage, $this->languageManager, $this->cache);
+    $this->entityQuery
+      ->expects($this->any())
+      ->method('accessCheck')
+      ->with(FALSE)
+      ->willReturn($this->entityQuery);
+
+    $this->entityQuery
+      ->expects($this->any())
+      ->method('sort')
+      ->with('id', 'DESC')
+      ->willReturn($this->entityQuery);
+
+    $this->entityQuery
+      ->expects($this->any())
+      ->method('range')
+      ->with(0, 1)
+      ->willReturn($this->entityQuery);
+
+    $this->entityTypeManager
+      ->expects($this->any())
+      ->method('getStorage')
+      ->with('path_alias')
+      ->willReturn($this->aliasStorage);
+
+    $this->aliasStorage
+      ->expects($this->any())
+      ->method('getQuery')
+      ->willReturn($this->entityQuery);
+
+    $this->appPathManager = new AppPathManager(
+      $this->aliasManager,
+      $this->entityTypeManager,
+      $this->appPathStorage,
+      $this->languageManager,
+      $this->cache
+    );
 
   }
 

--- a/docroot/modules/custom/app_module/tests/src/Unit/AppPathManagerUpdatePathDataTest.php
+++ b/docroot/modules/custom/app_module/tests/src/Unit/AppPathManagerUpdatePathDataTest.php
@@ -102,14 +102,24 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
    */
   public function testGoodEntityNoAlias() {
 
-    $this->aliasStorage
+    $this->entityQuery
+      ->expects($this->at(1))
+      ->method('condition')
+      ->with('path', '/node/22', '=', NULL);
+
+    $this->entityQuery
+      ->expects($this->at(2))
+      ->method('condition')
+      ->with('langcode', 'en', '=', NULL);
+
+    $this->entityQuery
       ->expects($this->exactly(1))
-      ->method('load')
-      ->with([
-        'source' => '/node/22',
-        'langcode' => 'en',
-      ])
-      ->willReturn(FALSE);
+      ->method('execute')
+      ->willReturn([]);
+
+    $this->aliasStorage
+      ->expects($this->never())
+      ->method('loadMultiple');
 
     // Setup Storage.
     // Expected Load looking for an existing app path.
@@ -139,7 +149,7 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
    * Positive test for normal action.
    *
    * This is the most complicated. The was no app path found, so
-   * we need to create it and not juust update.
+   * we need to create it and not just update.
    */
   public function testGoodEntityNew() {
 
@@ -153,19 +163,29 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
       'langcode' => 'en',
     ];
 
-    // Alias storage to lookup alias.
+    $this->entityQuery
+      ->expects($this->at(1))
+      ->method('condition')
+      ->with('path', '/node/22', '=', NULL)
+      ->willReturn($this->entityQuery);
+
+    $this->entityQuery
+      ->expects($this->at(2))
+      ->method('condition')
+      ->with('langcode', 'en', '=', NULL)
+      ->willReturn($this->entityQuery);
+
+    $this->entityQuery
+      ->expects($this->exactly(1))
+      ->method('execute')
+      ->willReturn([123]);
+
     $this->aliasStorage
       ->expects($this->exactly(1))
-      ->method('load')
-      ->with([
-        'source' => '/node/22',
-        'langcode' => 'en',
-      ])
+      ->method('loadMultiple')
+      ->with([123])
       ->willReturn([
-        'pid' => 123,
-        'source' => '/node/22',
-        'alias' => '/test-alias',
-        'langcode' => 'en',
+        $this->getPathAliasMock('123', '/node/22', '/test-alias', 'en'),
       ]);
 
     // Setup Storage.
@@ -239,19 +259,27 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
       'langcode' => 'en',
     ];
 
-    // Alias storage to lookup alias.
+    $this->entityQuery
+      ->expects($this->at(1))
+      ->method('condition')
+      ->with('path', '/node/22', '=', NULL);
+
+    $this->entityQuery
+      ->expects($this->at(2))
+      ->method('condition')
+      ->with('langcode', 'en', '=', NULL);
+
+    $this->entityQuery
+      ->expects($this->exactly(1))
+      ->method('execute')
+      ->willReturn([123]);
+
     $this->aliasStorage
       ->expects($this->exactly(1))
-      ->method('load')
-      ->with([
-        'source' => '/node/22',
-        'langcode' => 'en',
-      ])
+      ->method('loadMultiple')
+      ->with([123])
       ->willReturn([
-        'pid' => 123,
-        'source' => '/node/22',
-        'alias' => '/test-alias',
-        'langcode' => 'en',
+        $this->getPathAliasMock('123', '/node22', '/test-alias', 'en'),
       ]);
 
     // Setup Storage.
@@ -378,7 +406,7 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
               return $app_module_field_list;
 
             default:
-              throw new Exception("Unknown field " . $field_name);
+              throw new \Exception("Unknown field " . $field_name);
           }
         }
       ));
@@ -410,7 +438,7 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
               return $app_module_data;
 
             default:
-              throw new Exception("Unknown field " . $field_name);
+              throw new \Exception("Unknown field " . $field_name);
           }
         }
       ));
@@ -486,7 +514,7 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
                 return $has_app_id;
 
               default:
-                throw new Exception("Unexpected field " . $field_name);
+                throw new \Exception("Unexpected field " . $field_name);
             }
           }
         )
@@ -494,6 +522,43 @@ class AppPathManagerUpdatePathDataTest extends AppPathManagerTestBase {
     }
 
     return $entity;
+  }
+
+  /**
+   * Helper method to get content entity for negative tests.
+   */
+  private function getPathAliasMock($id, $path, $alias, $langcode) {
+    $pathAliasMock = $this->createMock('Drupal\path_alias\PathAliasInterface');
+    $pathAliasMock
+      ->expects($this->any())
+      ->method('id')
+      ->willReturn($id);
+
+    $pathAliasMock
+      ->expects($this->any())
+      ->method('getPath')
+      ->willReturn($path);
+
+    $pathAliasMock
+      ->expects($this->any())
+      ->method('getAlias')
+      ->willReturn($alias);
+
+    $langcodeMock = $this->getMockBuilder('\Drupal\Core\Field\FieldItemListInterface')
+      ->disableOriginalConstructor()
+      ->getMock();
+    $langcodeMock->expects($this->any())
+      ->method('__get')
+      ->with('value')
+      ->willReturn($langcode);
+
+    $pathAliasMock
+      ->expects($this->any())
+      ->method('get')
+      ->with('langcode')
+      ->willReturn($langcodeMock);
+
+    return $pathAliasMock;
   }
 
 }

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/cgov_blog.services.yml
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/cgov_blog.services.yml
@@ -2,11 +2,10 @@ services:
   cgov_blog.blog_manager:
     class: Drupal\cgov_blog\Services\BlogManager
     arguments:
-      - '@entity.query'
       - '@entity.repository'
       - '@entity_type.manager'
       - '@current_route_match'
-      - '@path.alias_manager'
+      - '@path_alias.manager'
       - '@language_manager'
       - '@request_stack'
   cgov_blog.twig_extensions:

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Services/BlogManager.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Services/BlogManager.php
@@ -5,9 +5,8 @@ namespace Drupal\cgov_blog\Services;
 use Drupal\Core\Entity\ContentEntityInterface;
 use Drupal\Core\Entity\EntityRepositoryInterface;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
-use Drupal\Core\Entity\Query\QueryFactory;
 use Drupal\Core\Language\LanguageManagerInterface;
-use Drupal\Core\Path\AliasManagerInterface;
+use Drupal\path_alias\AliasManagerInterface;
 use Drupal\Core\Routing\RouteMatchInterface;
 use Drupal\Core\TypedData\TranslatableInterface;
 use Symfony\Component\HttpFoundation\RequestStack;
@@ -16,13 +15,6 @@ use Symfony\Component\HttpFoundation\RequestStack;
  * Blog Manager Service.
  */
 class BlogManager implements BlogManagerInterface {
-
-  /**
-   * An entity query.
-   *
-   * @var Drupal\Core\Entity\Query\QueryFactory
-   */
-  protected $entityQuery;
 
   /**
    * The entity repository.
@@ -48,7 +40,7 @@ class BlogManager implements BlogManagerInterface {
   /**
    * The path alias manager.
    *
-   * @var \Drupal\Core\Path\AliasManagerInterface
+   * @var \Drupal\path_alias\AliasManagerInterface
    */
   protected $aliasManager;
 
@@ -70,15 +62,13 @@ class BlogManager implements BlogManagerInterface {
   /**
    * Constructor for BlogManager object.
    *
-   * @param \Drupal\Core\Entity\Query\QueryFactory $entity_query
-   *   An entity query.
    * @param \Drupal\Core\Entity\EntityRepositoryInterface $entity_repository
    *   The entity repository.
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
    *   The entity type manager service.
    * @param \Drupal\Core\Routing\RouteMatchInterface $route_matcher
    *   The route matcher.
-   * @param \Drupal\Core\Path\AliasManagerInterface $alias_manager
+   * @param \Drupal\path_alias\AliasManagerInterface $alias_manager
    *   The path alias manager.
    * @param \Drupal\Core\Language\LanguageManagerInterface $language_manager
    *   The language manager.
@@ -86,7 +76,6 @@ class BlogManager implements BlogManagerInterface {
    *   An HTTP request.
    */
   public function __construct(
-    QueryFactory $entity_query,
     EntityRepositoryInterface $entity_repository,
     EntityTypeManagerInterface $entity_type_manager,
     RouteMatchInterface $route_matcher,
@@ -94,7 +83,6 @@ class BlogManager implements BlogManagerInterface {
     LanguageManagerInterface $language_manager,
     RequestStack $request_stack
   ) {
-    $this->entityQuery = $entity_query;
     $this->entityRepository = $entity_repository;
     $this->entityTypeManager = $entity_type_manager;
     $this->routeMatcher = $route_matcher;
@@ -371,7 +359,7 @@ class BlogManager implements BlogManagerInterface {
    *   Content type or bundle.
    */
   public function getNodesByPostedDateAsc($type) {
-    $query = $this->entityQuery->get('node');
+    $query = $this->entityTypeManager->getStorage('node')->getQuery();
     $query->condition('status', 1);
     $query->condition('type', $type);
     $query->condition('langcode', $this->getCurrentLang());
@@ -387,7 +375,7 @@ class BlogManager implements BlogManagerInterface {
    *   Content type or bundle.
    */
   public function getNodesByPostedDateDesc($type) {
-    $query = $this->entityQuery->get('node');
+    $query = $this->entityTypeManager->getStorage('node')->getQuery();
     $query->condition('status', 1);
     $query->condition('type', $type);
     $query->condition('langcode', $this->getCurrentLang());

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module
@@ -77,7 +77,9 @@ EOMarkup;
   $result = $query->execute();
   if (count($result)) {
     $eid = array_shift($result);
-    $block = entity_load('block_content', $eid);
+    $block = \Drupal::entityTypeManager()
+      ->getStorage('block_content')
+      ->load($eid);
     $title = $block->info->value . " - " . $langcode;
     // Create translation if it doesn't already exist.
     if (!$block->hasTranslation($langcode)) {
@@ -87,7 +89,7 @@ EOMarkup;
         'body' => ['value' => $public_use[$langcode], 'format' => 'full_html'],
       ]);
       $block_translated->save();
-      drupal_set_message("Added Public Use block_content: $title");
+      \Drupal::messenger()->addStatus("Added Public Use block_content: $title");
     }
   }
 }

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.services.yml
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.services.yml
@@ -18,7 +18,7 @@ services:
     class: Drupal\cgov_core\Services\CgovNavigationManager
     arguments:
       - '@path.current'
-      - '@path.alias_manager'
+      - '@path_alias.manager'
       - '@entity_type.manager'
       - '@entity_field.manager'
       - '@logger.channel.cgov_core'
@@ -46,7 +46,7 @@ services:
       - { name: event_subscriber }
   cgov_core.url_path_processor:
     class: Drupal\cgov_core\Plugin\PathProcessor\UrlProcessor
-    arguments: ['@path.alias_manager', '@language_manager', '@service_container']
+    arguments: ['@path_alias.manager', '@language_manager', '@service_container']
     tags:
       - { name: path_processor_outbound, priority: 600 }
   cgov_core.config_overrider:

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/PathProcessor/UrlProcessor.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/PathProcessor/UrlProcessor.php
@@ -8,7 +8,7 @@ use Drupal\Core\PathProcessor\OutboundPathProcessorInterface;
 use Drupal\language\Entity\ConfigurableLanguage;
 use Drupal\Core\Language\LanguageManagerInterface;
 use Drupal\Core\Language\LanguageInterface;
-use Drupal\Core\Path\AliasManagerInterface;
+use Drupal\path_alias\AliasManagerInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 
 /**
@@ -39,14 +39,14 @@ class UrlProcessor implements OutboundPathProcessorInterface {
   /**
    * An alias manager for looking up the system path.
    *
-   * @var \Drupal\Core\Path\AliasManagerInterface
+   * @var \Drupal\path_alias\AliasManagerInterface
    */
   protected $aliasManager;
 
   /**
    * Constructor.
    *
-   * @param \Drupal\Core\Path\AliasManagerInterface $aliasManager
+   * @param \Drupal\path_alias\AliasManagerInterface $aliasManager
    *   An alias manager for looking up the system path.
    * @param \Drupal\Core\Language\LanguageManagerInterface $languageManager
    *   Language manager.

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Services/CgovNavigationManager.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Services/CgovNavigationManager.php
@@ -2,7 +2,7 @@
 
 namespace Drupal\cgov_core\Services;
 
-use Drupal\Core\Path\AliasManagerInterface;
+use Drupal\path_alias\AliasManagerInterface;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Entity\EntityFieldManagerInterface;
 use Drupal\Core\Entity\EntityRepository;
@@ -75,7 +75,7 @@ class CgovNavigationManager {
   /**
    * Path alias manager.
    *
-   * @var \Drupal\Core\Path\AliasManagerInterface
+   * @var \Drupal\path_alias\AliasManagerInterface
    */
   protected $pathAliasManager;
 

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_event/src/Controller/ICalendarController.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_event/src/Controller/ICalendarController.php
@@ -19,9 +19,21 @@ use Drupal\Core\File\FileSystem;
  */
 class ICalendarController extends ControllerBase {
 
+  /**
+   * {@inheritdoc}
+   */
   protected $entity;
+  /**
+   * {@inheritdoc}
+   */
   protected $currentUser;
+  /**
+   * {@inheritdoc}
+   */
   public $request;
+  /**
+   * {@inheritdoc}
+   */
   protected $file;
 
   /**
@@ -48,7 +60,7 @@ class ICalendarController extends ControllerBase {
    */
   public static function create(ContainerInterface $container) {
     return new static(
-      $container->get('entity.manager')->getStorage('node'),
+      $container->get('entity_type.manager')->getStorage('node'),
       $container->get('current_user'),
       $container->get('request_stack'),
       $container->get('file_system')
@@ -108,7 +120,7 @@ class ICalendarController extends ControllerBase {
     $filename = 'cal-' . $nid . '.ics';
     $uri = 'public://' . $filename;
     $content = $vCalendar->render();
-    $file = file_save_data($content, $uri, FILE_EXISTS_REPLACE);
+    $file = file_save_data($content, $uri, EXISTS_REPLACE);
     if (empty($file)) {
       return new Response(
         'iCalendar Error, please contact the System Administrator'

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_migration/cgov_migration.install
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_migration/cgov_migration.install
@@ -11,9 +11,9 @@
  * Removes stale migration configs during uninstall.
  */
 function cgov_migration_uninstall() {
-  $query = db_select('config', 'c')
+  $query = \Drupal::database()->select('config', 'c')
     ->fields('c', ['name'])
-    ->condition('name', db_like('migrate_plus.') . '%', 'LIKE')
+    ->condition('name', \Drupal::database()->escapeLike('migrate_plus.') . '%', 'LIKE')
     ->execute();
 
   $config_names = $query->fetchAll();

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_redirect_manager/src/CgovImporterService.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_redirect_manager/src/CgovImporterService.php
@@ -10,14 +10,20 @@ use Drupal\Core\Language\LanguageManagerInterface;
 use Psr\Log\LoggerInterface;
 
 /**
- * Class CGovImporterService.
+ * Class CgovImporterService.
  *
  * @package Drupal\cgov_redirect_manager
  */
-class CGovImporterService {
+class CgovImporterService {
 
+  /**
+   * {@inheritdoc}
+   */
   public static $messages = [];
 
+  /**
+   * {@inheritdoc}
+   */
   public static $options = [];
 
   /**
@@ -81,12 +87,12 @@ class CGovImporterService {
     if ($options['suppress_messages'] != 1) {
       // Messaging/logging is separated out in case we want to suppress these.
       foreach (self::$messages['warning'] as $warning) {
-        drupal_set_message($warning, 'warning');
+        \Drupal::messenger()->addWarning($warning);
       }
     }
 
     if (empty($data)) {
-      drupal_set_message(t('The uploaded file contains no rows with compatible redirect data. No redirects have imported. Compare your file to <a href=":sample">this sample data.</a>', [':sample' => '/' . drupal_get_path('module', 'path_redirect_import') . '/redirect-example-file.csv']), 'warning');
+      \Drupal::messenger()->addWarning(t('The uploaded file contains no rows with compatible redirect data. No redirects have imported. Compare your file to <a href=":sample">this sample data.</a>', [':sample' => '/' . drupal_get_path('module', 'path_redirect_import') . '/redirect-example-file.csv']));
     }
     else {
       if (PHP_SAPI == 'cli' && function_exists('drush_main')) {
@@ -99,7 +105,7 @@ class CGovImporterService {
         // This case should never happen...
       }
     }
-    drupal_set_message(t('Successfully imported "@count" items.', ['@count' => $count]));
+    \Drupal::messenger()->addStatus(t('Successfully imported "@count" items.', ['@count' => $count]));
   }
 
   /**
@@ -112,7 +118,7 @@ class CGovImporterService {
     else {
       $message = t('Finished with an error.');
     }
-    drupal_set_message($message, 'status');
+    \Drupal::messenger()->addStatus($message);
   }
 
   /**
@@ -213,7 +219,7 @@ class CGovImporterService {
     while ($line = self::readExpandLine($f, $options['delimiter'], $options['status_code'], $options['language'])) {
       $line_no++;
       if ($line_no == 1 && !$options['no_headers']) {
-        drupal_set_message(t('Skipping the header row.'));
+        \Drupal::messenger()->addStatus(t('Skipping the header row.'));
         continue;
       }
 
@@ -470,7 +476,7 @@ class CGovImporterService {
     $hash = Redirect::generateHash($path, $query, $row['language']);
 
     // Search for duplicate.
-    $redirects = \Drupal::entityManager()
+    $redirects = \Drupal::entityTypeManager()
       ->getStorage('redirect')
       ->loadByProperties(['hash' => $hash]);
     if (!empty($redirects)) {

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_redirect_manager/src/Form/RedirectImport.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_redirect_manager/src/Form/RedirectImport.php
@@ -15,6 +15,9 @@ use Drupal\Core\File\FileSystem;
  */
 class RedirectImport extends FormBase {
 
+  /**
+   * {@inheritdoc}
+   */
   protected $filesystem;
 
   /**
@@ -118,12 +121,12 @@ class RedirectImport extends FormBase {
       $form_state->setErrorByName('file', $this->t('No file was found.'));
       return;
     }
-    drupal_set_message($this->t("Successfully uploaded redirect file."));
+    $this->messenger()->addStatus($this->t("Successfully uploaded redirect file."));
 
     ini_set('auto_detect_line_endings', TRUE);
     // Don't do anything if no valid file.
     if (!isset($this->file)) {
-      drupal_set_message($this->t('No valid file was found. No redirects will be imported.'), 'warning');
+      $this->messenger()->addWarning($this->t('No valid file was found. No redirects will be imported.'));
       return;
     }
 

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/src/Plugin/Validation/Constraint/UniqueCgovUrlConstraintValidator.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/src/Plugin/Validation/Constraint/UniqueCgovUrlConstraintValidator.php
@@ -7,6 +7,7 @@ use Symfony\Component\Validator\ConstraintValidator;
 use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Field\FieldItemListInterface;
 use Drupal\Core\Field\EntityReferenceFieldItemListInterface;
 
@@ -22,14 +23,30 @@ class UniqueCgovUrlConstraintValidator extends ConstraintValidator implements Co
    *
    * @var \Symfony\Component\Validator\Context\ExecutionContextInterface
    */
-
   protected $context;
+
+  /**
+   * Entity Type Manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
 
   /**
    * {@inheritdoc}
    */
   public static function create(ContainerInterface $container) {
-    return new static($container->get('entity.manager')->getStorage('node'));
+    return new static($container->get('entity_type.manager'));
+  }
+
+  /**
+   * Constructs an UniqueCgovUrlConstraintValidator object.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->entityTypeManager = $entity_type_manager;
   }
 
   /**
@@ -115,7 +132,10 @@ class UniqueCgovUrlConstraintValidator extends ConstraintValidator implements Co
 
       // Query for the conditions.
       // The id could be NULL, cast it to 0 in that case.
-      $value_taken = (bool) \Drupal::entityQuery($entity_type_id)
+      $query = $this->entityTypeManager
+        ->getStorage($entity_type_id)
+        ->getQuery();
+      $value_taken = (bool) $query
         ->condition($id_key, (int) $entity->id(), '<>')
         ->condition($field_name, $value)
         ->notExists('field_pretty_url')
@@ -158,7 +178,10 @@ class UniqueCgovUrlConstraintValidator extends ConstraintValidator implements Co
 
     // Query for the conditions.
     // The id could be NULL, cast it to 0 in that case.
-    $value_taken = (bool) \Drupal::entityQuery($entity_type_id)
+    $query = $this->entityTypeManager
+      ->getStorage($entity_type_id)
+      ->getQuery();
+    $value_taken = (bool) $query
       ->condition($id_key, (int) $entity->id(), '<>')
       ->condition($section_field_name, $section_value)
       ->condition($pretty_url_field_name, $pretty_url_value)

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_syndication/cgov_syndication.module
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_syndication/cgov_syndication.module
@@ -61,7 +61,7 @@ EOMarkup;
   $result = $query->execute();
   if (count($result)) {
     $eid = array_shift($result);
-    $block = entity_load('block_content', $eid);
+    $block = \Drupal::entityTypeManager()->getStorage('block_content')->load($eid);
     $title = $block->info->value . " - " . $langcode;
     // Create translation if it doesn't already exist.
     if (!$block->hasTranslation($langcode)) {
@@ -71,7 +71,7 @@ EOMarkup;
         'field_raw_html' => ['value' => $syndication_message[$langcode], 'format' => 'raw_html'],
       ]);
       $block_translated->save();
-      drupal_set_message("Added Syndication Message block_content: $title");
+      \Drupal::messenger()->addStatus('Added Syndication Message block_content: ' . $title);
     }
   }
 }

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_vocab_manager/src/Manager/CgovVocabManager.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_vocab_manager/src/Manager/CgovVocabManager.php
@@ -42,8 +42,9 @@ class CgovVocabManager {
         $first_element = FALSE;
       }
       else {
+
         $parent_link_url = Url::fromRoute('cgov_vocab_manager.overview_onelevel_form', [
-          'taxonomy_vocabulary' => $parent->getVocabularyId(),
+          'taxonomy_vocabulary' => $parent->bundle(),
           'parent_tid' => $parent->id(),
         ]);
       }
@@ -53,7 +54,7 @@ class CgovVocabManager {
     }
     // Add Root in so users can get back to main overview page.
     $parents_links[] = Link::fromTextAndUrl('Level 0 (Root)', Url::fromRoute('cgov_vocab_manager.overview_onelevel_form', [
-      'taxonomy_vocabulary' => $term->getVocabularyId(),
+      'taxonomy_vocabulary' => $term->bundle(),
       'parent_tid' => 0,
     ]));
     // Revert the order of the Url elements.
@@ -73,11 +74,11 @@ class CgovVocabManager {
    */
   public function getChildrenLink(Term $term, TermStorageInterface $storageController, RendererInterface $renderer) {
     // Count the number of children of the current term.
-    $children_count = count($storageController->loadChildren($term->id(), $term->getVocabularyId()));
+    $children_count = count($storageController->loadChildren($term->id(), $term->bundle()));
 
     // Generate the link.
     $link_manage_children_url = Url::fromRoute('cgov_vocab_manager.overview_onelevel_form', [
-      'taxonomy_vocabulary' => $term->getVocabularyId(),
+      'taxonomy_vocabulary' => $term->bundle(),
       'parent_tid' => $term->id(),
     ]);
     $link_manage_children_url_text = $this->formatPlural($children_count, '1 child', '@count children');

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/cgov_yaml_content.services.yml
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/cgov_yaml_content.services.yml
@@ -1,7 +1,7 @@
 services:
   cgov_yaml_content.event_subscriber:
     class: Drupal\cgov_yaml_content\Service\CgovYamlContentEventSubscriber
-    arguments: ['@theme.manager', '@entity.query', '@entity_type.manager', '@token', '@event_dispatcher']
+    arguments: ['@theme.manager', '@entity_type.manager', '@file_system', '@token', '@event_dispatcher']
     tags:
       - {name: event_subscriber}
   cgov_yaml_content.yaml_entity_embed_processor:

--- a/docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/src/Service/CgovYamlContentEventSubscriber.php
+++ b/docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/src/Service/CgovYamlContentEventSubscriber.php
@@ -9,8 +9,8 @@ use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Drupal\block\Entity\Block;
 use Drupal\paragraphs\Entity\Paragraph;
 use Drupal\Core\Theme\ThemeManagerInterface;
-use Drupal\Core\Entity\Query\QueryFactory;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\File\FileSystemInterface;
 use Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher;
 use Drupal\Component\Render\PlainTextOutput;
 use Drupal\Core\Utility\Token;
@@ -31,18 +31,18 @@ class CgovYamlContentEventSubscriber implements EventSubscriberInterface {
   protected $themeManager;
 
   /**
-   * Drupal Entity Query Factory.
-   *
-   * @var \Drupal\Core\Entity\Query\QueryFactory
-   */
-  protected $entityQueryFactory;
-
-  /**
    * Drupal Entity Type Manager.
    *
    * @var \Drupal\Core\Entity\EntityTypeManagerInterface
    */
   protected $entityTypeManager;
+
+  /**
+   * Drupal File System.
+   *
+   * @var \Drupal\Core\File\FileSystemInterface
+   */
+  protected $fileSystem;
 
   /**
    * Drupal token service.
@@ -70,10 +70,10 @@ class CgovYamlContentEventSubscriber implements EventSubscriberInterface {
    *
    * @param \Drupal\Core\Theme\ThemeManagerInterface $themeManager
    *   Theme Manager.
-   * @param \Drupal\Core\Entity\Query\QueryFactory $entityQueryFactory
-   *   Query factory.
    * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
    *   Drupal Entity Type Manager.
+   * @param \Drupal\Core\File\FileSystemInterface $fileSystem
+   *   Drupal file system wrapper.
    * @param \Drupal\Core\Utility\Token $token
    *   Drupal token service.
    * @param \Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher $dispatcher
@@ -81,14 +81,14 @@ class CgovYamlContentEventSubscriber implements EventSubscriberInterface {
    */
   public function __construct(
     ThemeManagerInterface $themeManager,
-    QueryFactory $entityQueryFactory,
     EntityTypeManagerInterface $entityTypeManager,
+    FileSystemInterface $fileSystem,
     Token $token,
     ContainerAwareEventDispatcher $dispatcher
     ) {
     $this->themeManager = $themeManager;
-    $this->entityQueryFactory = $entityQueryFactory;
     $this->entityTypeManager = $entityTypeManager;
+    $this->fileSystem = $fileSystem;
     $this->token = $token;
     $this->eventDispatcher = $dispatcher;
   }
@@ -198,7 +198,9 @@ class CgovYamlContentEventSubscriber implements EventSubscriberInterface {
     $fieldReferenceArray = [];
     $entity_type = $value['#process']['args'][0];
     $args = $value['#process']['args'][1];
-    $query = $this->entityQueryFactory->get($entity_type);
+    $query = $this->entityTypeManager
+      ->getStorage($entity_type)
+      ->getQuery();
     foreach ($args as $property => $value) {
       $query->condition($property, $value);
     }
@@ -257,10 +259,10 @@ class CgovYamlContentEventSubscriber implements EventSubscriberInterface {
       }
 
       // Create the destination directory if it does not already exist.
-      file_prepare_directory($destination, FILE_CREATE_DIRECTORY);
+      $this->fileSystem->prepareDirectory($destination, FileSystemInterface::CREATE_DIRECTORY);
 
       // Save the file data or return an existing file.
-      $file = file_save_data($fileData, $destination . $filename, FILE_EXISTS_REPLACE);
+      $file = file_save_data($fileData, $destination . $filename, FileSystemInterface::EXISTS_REPLACE);
 
       // Use the newly created file id as the value.
       $processConfig['target_id'] = $file->id();

--- a/docroot/profiles/custom/cgov_site/src/Form/CgovAlterForm.php
+++ b/docroot/profiles/custom/cgov_site/src/Form/CgovAlterForm.php
@@ -53,7 +53,7 @@ class CgovAlterForm extends AssignmentFormBase {
     $this->currentBundle->setAssignmentSettings(self::METHOD_ID, $settings)->save();
 
     $this->setRedirect($form_state);
-    drupal_set_message($this->t('Package assignment configuration saved.'));
+    $this->messenger()->addMessage($this->t('Package assignment configuration saved.'));
   }
 
 }

